draft-ietf-mls-protocol.md

```diff
@@ -1814,19 +1823,19 @@ The client verifies the validity of a LeafNode using the following steps:
 
 * Verify the `lifetime` field:
   * When validating a downloaded KeyPackage, the current time MUST be within the
-    `lifetime` range.  A KeyPackage that is expired or not yet valid MUST NOT be
+    `lifetime` range.  A ClientInfo that is expired or not yet valid MUST NOT be
```

Comment:
Should this be LeafNode, to be consistent with the above?

Reply:
There was one place in the KeyPackage section where we were talking about lifetimes. In that case, I think the lifetime originally referred to the lifetime of the KeyPackage.